src/main/java/it/pagopa/pn/deliverypush/dto/timeline/details/BaseRegisteredLetterDetailsInt.java

```diff
@@ -16,6 +16,7 @@ public class BaseRegisteredLetterDetailsInt implements RecipientRelatedTimelineE
     protected int recIndex;
     protected PhysicalAddressInt physicalAddress;
     protected String foreignState;
+    protected String requestId;
```

Comment:
rinominare in prepareRequestId

Reply:
resolved with commit ac0d3f71ccdd88155587af92c33c47243ad239e1